astronomer/providers/snowflake/hooks/snowflake.py

```diff
@@ -91,7 +91,9 @@ def check_query_output(self, query_ids: List[str]) -> None:
                     self.log.info("Rows affected: %s", cur.rowcount)
                     self.log.info("Snowflake query id: %s", query_id)
 
-    async def get_query_status(self, query_ids: List[str]) -> Dict[str, Union[str, List[str]]]:
+    async def get_query_status(
+        self, query_ids: List[str], poll_intervals: float
```

Comment:
This should be poll_interval and not poll_intervals as it is as single value

Reply:
Got it